django/contrib/postgres/search.py

```diff
@@ -208,7 +208,9 @@ class SearchRank(Func):
     function = 'ts_rank'
     output_field = FloatField()
 
-    def __init__(self, vector, query, weights=None):
+    def __init__(self, vector, query, weights=None, cover_density=False):
+        if cover_density:
+            self.function = 'ts_rank_cd'
```

Comment:
Minor but I'd move this control flow block after the `weights` one to match the args order.

Reply:
Makes sense 👍 . Had a though that it might be good to set `self.function` as early as possible.